CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
+
+Web3
+
+- Added Solana Swap integration into existing swap UI using Jupiter as the DEX aggregator ([#6847](https://github.com/brave/brave-ios/issues/6847))
+- Added message for Submitted/Completed state change after a transaction is submitted. ([#6960](https://github.com/brave/brave-ios/issues/6960))
+- Added UI to display ‘signed’ status for Ethereum transactions ([#6606](https://github.com/brave/brave-ios/issues/6606))
+- Added support for sending to ENS wallet address ([#5787](https://github.com/brave/brave-ios/issues/5787) )
+- Added auto-complete when entering token address for custom Solana assets  ([#6990](https://github.com/brave/brave-ios/issues/6990))
+- Fixed wallet issue where the Original Password field was shown when adding Solana Account ([#7175](https://github.com/brave/brave-ios/issues/7175))
+- Fixed reporting Brave wallet usage stats ([#7143](https://github.com/brave/brave-ios/issues/7143))
+- Fixed issue adding a custom network via panel prompt ([#6965](https://github.com/brave/brave-ios/issues/6965))
+- Fixed issue with ERC721 NFTs with different token identifiers showing incorrect balance ([#6954](https://github.com/brave/brave-ios/issues/6954))
+- Added support for Solana Name Service (SNS) navigation in app ([#6706](https://github.com/brave/brave-ios/issues/6706))
+
+General UI
+
+- Dropped support for iOS 14 devices ([#6695](https://github.com/brave/brave-ios/issues/6695))
+- Added an enhancement where only HTTP/HTTPS URLs are loaded when using QR scanner in the URL box. ([#6997](https://github.com/brave/brave-ios/issues/6997))
+- Added capability to permanently delete sync accounts ([#6820](https://github.com/brave/brave-ios/issues/6829))
+- Added a promotional callout for VPN subscribers for linking their receipt on brave.com which will enable use of cross-platform VPN ([#6164](https://github.com/brave/brave-ios/issues/6164))
```

Comment:
linking their receipt ->  linking their VPN receipt

or can be reworded to 

Added promotional callout for VPN cross-platform linking on account.brave.com